README.md

```diff
@@ -10,42 +10,60 @@ Install [Terraform](https://www.terraform.io) 0.12. On macOS, this is easy with
 
-Next, **make a plan** to find out how it will affect the current state of the system:
+To make changes in a workspace, first `cd` into the workspace's directory and run `terraform init` to pull down dependencies. Then, make your changes to the Terraform configuration files with your text editor.
+
+You can **make a plan** to find out how your changes will affect the current state of the system:
 
 ```sh
 make plan
 ```
 
-Once you're satisfied with Terraform's plan, commit your work & make a pull request. After your pull request is reviewed and merged, you can then **apply your change** to update the actual infrastructure. Terraform Cloud will make your changes, update the remote state, and ensure nobody else makes any changes until you're done.
+Once you're satisfied with Terraform's plan for your changes, commit your work & make a pull request. Your pull request will automatically run a plan for all workspaces (even if they're not affected by your change).
 
-See Terraform's [Getting Started guide](https://www.terraform.io/intro/getting-started/build.html) & [documentation](https://www.terraform.io/docs/index.html) for more details.
+### Apply
+
+After your pull request is reviewed and merged, you can then **apply your change** to update the actual infrastructure. Terraform Cloud will make your changes, update the remote state, and ensure nobody else makes any changes until you're done.
+
+To apply pending changes to a workspace, visit [Terraform Cloud](https://app.terraform.io/app/dosomething/workspaces) and open the latest run for the workspace you want to modify. Review the plan & then choose "Confirm & Apply" to make the change.
 
 ## Security Vulnerabilities
 
-We take security very seriously. Any vulnerabilities in our infrastructure should be reported to [security@dosomething.org](mailto:security@dosomething.org),
+We take security very seriously. Any vulnerabilities should be reported to [security@dosomething.org](mailto:security@dosomething.org),
 and will be promptly addressed. Thank you for taking the time to responsibly disclose any issues you find.
 
+## References
+- [Terraform "Getting Started" Tutorial](https://www.terraform.io/intro/getting-started/build.html) - a "step by step" tutorial on Terraform basics
+- [Terraform Configuration Language Reference](https://www.terraform.io/docs/configuration/index.html) – look here for syntax for writing config!
+- [Terraform AWS Provider](https://www.terraform.io/docs/providers/aws/) - API documentation for `aws_` resources
+- [Terraform Fastly Provider](https://www.terraform.io/docs/providers/fastly/) - API documentation for `fastly_` resources
+- [Terraform Heroku Provider](https://www.terraform.io/docs/providers/heroku/) - API documentation for `heroku_` resources
+- [Serverless Guide](https://github.com/DoSomething/infrastructure/blob/master/docs/serverless-guide.md) - how to use our "serverless" modules
```

Comment:
I was thinking this might be a helpful section to add to our README template - links to tutorials or documentation that might be relevant when onboarding to or working in an application! I always find myself going back to these same pages when working in Terraform & they're not all super easy to find!